packages/sui-studio/src/components/style/index.js

```diff
@@ -19,8 +19,12 @@ export default class Style extends Component {
     this._linkElement = createLinkElement()
   }
 
+  componentWillUnmount () {
+    const lastIndex = document.styleSheets.length - 1
+    document.styleSheets[lastIndex].disabled = true
+  }
```

Comment:
Maybe you can use an ID en the tag, to know always which one is the correct Style Tag. That solution can be a little fragile.